tests/components/ipp/test_init.py

```diff
@@ -37,10 +36,8 @@ async def test_load_unload_config_entry(
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
 
-    assert mock_config_entry.entry_id in hass.data[DOMAIN]
```

Comment:
Same comment here. You might want to test that the coordinator is properly stored in `mock_config_entry.runtime_data`.

Reply:
sounds reasonable 👍